index.html

```diff
@@ -2,9 +2,130 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Business Website Assignment</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./styles/styles.css">
 </head>
 <body>
-
+    <header>
+        <div class="outer-container">
+            <nav class="inner-container-large-nav">
+                <ul>
+                    <li>Case Studies</li>
+                    <li>Services</li>
+                    <li>Contact Us</li>
+                </ul>
+            </nav>
+        </div>
+        <div class="outer-container">
+            <div class="inner-container-small introblock">
+                <h1>Your website is an investment in the  success of your business</h1>
+                <p>We are here to help you succeed</p>
+                <button>LET'S WORK TOGETHER</button>
+                <img src="assets/squiggle_line_stripe.svg" alt="------" />
```

Comment:
Wanneer je een image hebt waar je geen alt tekst voor kunt bedenken omdat het eigenlijk geen afbeelding is voor de gebruiker, dan kun je beter de image implementeren met CSS i.p.v. met een img element.